### ISSUE TITLE
Configurate doHover floating window size

### ISSUE THREAD

**Issue by ytian81** [neoclide/coc.nvim#2984]
I am trying to use the following to configurate the hover float window but the window size doesn't seem to be able to change even after `CocRebuild` and `CocRestart`
```
    "hover.floatMaxWidth": 200,
    "hover.floatMaxHeight": 200,

```

_Originally posted by @ytian81 in https://github.com/neoclide/coc.nvim/issues/2979#issuecomment-804443305_

**Comment by Avi-D-coder** [comment]
Can reproduce.
My hover float is now 80 characters wide, the setting is ignored.

**Comment by chemzqm** [comment]
It's limited to 80.

**Comment by ytian81** [comment]
Is it possible to set the width from 80 to some other values?

**Comment by chemzqm** [comment]
Change this line https://github.com/neoclide/coc.nvim/blob/master/autoload/coc/float.vim#L849

**Comment by ytian81** [comment]
It would be ideal to set this value to some large value like 500 or 1000 and then use `"hover.floatMaxWidth": 80` to limit the width instead.

**Comment by chemzqm** [comment]
Fixed by cc1ad566
